lib/perl/Genome/InstrumentData/Command/Import/WorkFlow/FastqsToBam.pm

```diff
@@ -57,6 +63,40 @@ sub execute {
     return 1;
 }
 
+sub _unarchive_fastqs_if_necessary {
+    my $self = shift;
+    $self->debug_message('Unarchive fastqs if necessary...');
+
+    my @new_fastq_paths;
+    for my $fastq_path ( $self->fastq_paths ) {
+        if ( $fastq_path !~ /\.gz$/ ) {
+            $self->debug_message('Unarchive not necessary for '.$fastq_path);
+            push @new_fastq_paths, $fastq_path;
+            next;
+        }
+        my $unarchived_fastq_path = $fastq_path;
+        $unarchived_fastq_path =~ s/\.gz$//;
+        $self->debug_message('Unarchiving: '.$fastq_path);
+        my $rv = eval{ Genome::Sys->shellcmd(cmd => "gunzip $fastq_path"); };
```

Comment:
I originally did use it, except it ignored the Archive::Extract::PREFER_BIN=1 and sucked a fastq into memory, killing the process. Hence shelling the shelling out. I forgot to remove it.